CHANGELOG.md

```diff
@@ -1,16 +1,12 @@
 # Change Log
 
-## 16.2.1 (unreleased)
+<!-- You should *NOT* be adding new change log entries to this file.
+     You should create a file in the news directory instead.
+     For helpful instructions, please see:
+     https://github.com/plone/plone.releaser/blob/master/ADD-A-NEWS-ITEM.rst
```

Comment:
This link is not completely accurate, as it does not include `.documentation` or `.docs`. Avoid using `.doc` because some editors, such as PyCharm, will attempt to open the file in an Office word processor. We need to document these differences.

Reply:
```suggestion
     https://6.dev-docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request
```

This PR needs to update and add documentation to [Create a pull request](https://6.dev-docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request).

> All pull requests must include a note under the (unreleased) version under the appropriate subheading of [CHANGELOG.md](https://github.com/plone/volto/blob/master/CHANGELOG.md). Do not edit already released versions.

The above should be changed to:

```
All pull requests must include a `towncrier` news item.
This is a file that is placed in the root of the repository directory at `/news`.
Its format must be `###.type`, where `###` is the referenced GitHub issue or pull request number, `.` is the literal extension delimiter, and `type` is one of the following strings.

-   `breaking` for breaking changes
-   `bugfix` for bug fixes
-   `documentation` for documentation
-   `feature` for new features
-   `internal` for internal changes
```

Reply:
@stevepiercy ok, I will amend the config. 
However, I just copied over the message from `plone.restapi`. Maybe we need to improve/fix all these messages across all Plone packages repos. /cc @mauritsvanrees

Reply:
Amended https://6.dev-docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request